docs/input-manifest/providers/cloudflare.md

```diff
@@ -1,16 +1,81 @@
 # Cloudflare
+Cloudflare provider requires `apiToken` token field in string format.
 
-For using your Cloudflare provider for DNS, you need to manually
+## DNS example
+```yaml
+providers:
+  cloudflare:
+    - name: cloudflare-1
+      apiToken: token
+```
 
-- [set up zone](https://developers.cloudflare.com/dns/zone-setups/)
-- Update your name servers for you domain with the NS records from the created DNS zone.
+## Create Cloudflare credentials
+You can create Cloudflare API token by following [this guide](https://developers.cloudflare.com/fundamentals/api/get-started/create-token/). The required permissions for the zone you want to use are:
 
-since Claudie does not support their dynamic creation.
+```bash
```

Comment:
I'm just curious, why is there DNS:Read and also DNS:Edit?

Reply:
Claudie needs read and write to manage dns records in CF hosted zones. Adding and deleting records requires edit privs.